### PR TITLE
[reliability] update filter table  for reliability

### PR DESCRIPTION
**Pull request by kongtiaowang**

### CODE REVIEW

File: modules/reliability/jsx/columnFormatter.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  // Create the mapping between rowHeaders and rowData in a row object.
+  var row = {};
+  rowHeaders.forEach(function(header, index) {
+    row[header] = rowData[index];
+}, this);
+
+ 
+    if(column === 'PSCID' ) {
+        if (row["Manual Swap"]==="yes")
+        {
```
Comment (driusan) [inline modules/reliability/jsx/columnFormatter.js]: I can't remember if JS will implicitly add a semicolon before the `{` here or not ( @alisterdev? @jstirling91?)

Regardless, the formatting of this whole block is off and hard to read. Can you run this file through eslint and fix the formatting?

Comment (alisterdev) [inline modules/reliability/jsx/columnFormatter.js]:
```js
if (stmnt)
{
 // stuff
}
```
@driusan This is valid format, but as you mentioned not standard in Loris and hard to read. ESLint would definetly be apreciated.

File: modules/reliability/jsx/columnFormatter.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  rowHeaders.forEach(function(header, index) {
+    row[header] = rowData[index];
+}, this);
+
+ 
+    if(column === 'PSCID' ) {
+        if (row["Manual Swap"]==="yes") {
+        return <td>{cell} <span className="error">manual</span> </td>;
+        }
+        if(row["Invalid"] === 'yes' ) {
```
Comment (alisterdev) [inline modules/reliability/jsx/columnFormatter.js]: This still doesn't look formatted. Please run ESLint.
From you loris root run `node_modules/.bin/eslint --fix modules/reliability/`

File: modules/reliability/jsx/columnFormatter.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+/* global hasWritePermission */
```
Comment (alisterdev) [inline modules/reliability/jsx/columnFormatter.js]: you don't need this like since you are not using hasWritePermission variable

**Comment by stellarxo** [comment]
Before:
screen shot 2016-09-01 at 10 06 39 am

After:
screen shot 2016-09-01 at 10 06 31 am

Cohort and Project aren't being shown properly

**Comment by kongtiaowang** [comment]
@stellarxo  
Could u test again? I have fixed the bug.
screen shot 2016-09-06 at 11 58 22 am

**Comment by stellarxo** [comment]
@kongtiaowang it's still not fixed for project - you have to turn it on in your config settings

image

**Comment by kongtiaowang** [comment]
@stellarxo 
The project column should not appear in the table ,got it.
What's the meaning of turn on config settings?

**Comment by kongtiaowang** [comment]
@stellarxo 
What's the meaning of "turn on config settings"?

**Comment by stellarxo** [comment]
Update this in your configuration module:

image

**Comment by kongtiaowang** [comment]
@stellarxo 
The project column is not right. I got it.

**Comment by stellarxo** [comment]
Reliable column should say "No" with a red cell background, not 0.00

image

**Comment by kongtiaowang** [comment]
@stellarxo 
screen shot 2016-09-08 at 3 25 33 pm

**Comment by stellarxo** [comment]
@kongtiaowang I think null score is okay because not all of them will have a score

**Comment by kongtiaowang** [comment]
@stellarxo  
If you have time pls test it. Thanks.

**Comment by alisterdev** [comment]
What Shen said ^

**Pull request closed without merging**